### PR TITLE
Add vararg constructor to InMemoryEntityLookup

### PR DESCRIPTION
**Pull request by JeroenDeDauw**
Noticed absence in
https://gerrit.wikimedia.org/r/c/mediawiki/extensions/WikibaseQualityConstraints/+/473719/1/tests/phpunit/Helper/ExceptionIgnoringEntityLookupTest.php

### CODE REVIEW

File: src/Lookup/InMemoryEntityLookup.php [hunk 1]
```diff
@@ -30,6 +30,15 @@ class InMemoryEntityLookup implements EntityLookup {
 	 */
 	private $exceptions = [];
 
+	/**
+	 * @param EntityDocument ...$entities
+	 */
+	public function __construct( ...$entities ) {
```
Comment (thiemowmde) [inline src/Lookup/InMemoryEntityLookup.php:36]: I'm somewhat surprised by this. I faced heavy backlash when I tried to introduce pretty much the exact same back then in https://github.com/wmde/WikibaseDataModel/pull/376 and https://github.com/wmde/WikibaseDataModel/pull/416. Not meant to block this, not even as a vote. I'm just curious what changed and what others think nowadays.

Comment (JeroenDeDauw) [reply src/Lookup/InMemoryEntityLookup.php:36]: What I am doing here is adding an array parameter using varargs. That is not the same as changing an existing parameter to support two input styles. The context is also different.

Comment (thiemowmde) [reply src/Lookup/InMemoryEntityLookup.php:36]: I would not call this an "array parameter". The `...` syntax as documented at http://php.net/manual/en/functions.arguments.php#functions.variable-arg-list does not allow callers to pass an array, but a "variable-length list of arguments". Which is what I wanted in these other pull requests.

**Comment by JeroenDeDauw** [comment]
@manicki can you get this simple change unstuck?

**Comment by mariushoch** [comment]
+1 Looks good to me

**Review by manicki — APPROVED** [review]

**Pull request merged**